[DOCS-9644] Standalone Error Tracking for Mobile (#26569)
DataDog/documentation: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/kotlin-multiplatform.md

Kotlin Multiplatform Monitoring is in Preview.
{{< /beta-callout >}}

> This page describes how to instrument your applications for both [Real User Monitoring (RUM)][1] and [Error Tracking][2] with the Flutter SDK. You can follow the steps below to instrument your applications for RUM (includes Error Tracking), or Error Tracking if you have purchased it as a standalone product.

Author: ```suggestion
This page describes how to instrument your applications for both [Real User Monitoring (RUM)][1] and [Error Tracking][2] with the Kotlin Multiplatform SDK. You can follow the steps below to instrument your applications for RUM (includes Error Tracking), or Error Tracking if you have purchased it as a standalone product.
```

### Specify application details in the UI

{{< tabs >}}
{{% tab "RUM" %}}

1. Navigate to [**Digital Experience** > **Add an Application**][1].
2. Select `Kotlin Multiplatform` as the application type and enter an application name to generate a unique Datadog application ID and client token.

> 3. To disable automatic user data collection for either client IP or geolocation data, use the toggles for those settings. For more information, see [Kotlin Multiplatform Data Collected][2].

Reviewer: ```suggestion
3. To disable automatic user data collection for client IP or geolocation data, use the toggles for those settings. For more information, see [Kotlin Multiplatform Data Collected][2].
```

[1]: https://app.datadoghq.com/rum/application/create
[2]: /real_user_monitoring/kotlin-multiplatform/data_collected/

{{% /tab %}}
{{% tab "Error Tracking" %}}

1. Navigate to [**Error Tracking** > **Settings** > **Browser and Mobile** > **Add an Application**][1].
2. Select `Kotlin Multiplatform` as the application type and enter an application name to generate a unique Datadog application ID and client token.

> 3. To disable automatic user data collection for either client IP or geolocation data, use the toggles for those settings. For more information, see [Kotlin Multiplatform Data Collected][2].

Reviewer: ```suggestion
3. To disable automatic user data collection for client IP or geolocation data, use the toggles for those settings. For more information, see [Kotlin Multiplatform Data Collected][2].
```

   {{< img src="real_user_monitoring/error_tracking/mobile-new-application.png" alt="Create an application for iOS in Datadog" style="width:90%;">}}

[1]: https://app.datadoghq.com/error-tracking/settings/setup/client
[2]: /real_user_monitoring/kotlin-multiplatform/data_collected/

{{% /tab %}}
{{< /tabs >}}

> To ensure the safety of your data, you must use a client token. If you used only [Datadog API keys][4] to configure the Datadog SDK, they would be exposed client-side in the Android application's APK byte code. 

Reviewer: ```suggestion
To ensure the safety of your data, you must use a client token. If you use only [Datadog API keys][4] to configure the Datadog SDK, they are exposed client-side in the Android application's APK byte code. 
```

### Sample RUM sessions

> To control the data your application sends to Datadog RUM, you can specify a sample rate for RUM sessions while [initializing the RUM feature][8] as a percentage between 0 and 100.

Reviewer: ```suggestion
To control the data your application sends to Datadog RUM, you can specify a sample rate for RUM sessions while [initializing the RUM feature][8]. This rate is a percentage between 0 and 100.
```

{{< partial name="whats-next/whats-next.html" >}}

> [1]:/real_user_monitoring/

Author: ```suggestion
[1]: /real_user_monitoring/
```
